README.md

```diff
@@ -42,13 +44,14 @@ services:
 ```
 
 2) Convert OpenCompose file to Kubernetes objects
+
 ```sh
 opencompose convert -f hello-nginx.yaml
 ```
-This will create two files in current directory - *deployment-helloworld.yaml* and *service-helloworld.yaml*.
-First file is a Kubernetes Deployment object and second is a Kubernetes Service.
 
-To deploy your application to Kubernetes cluster just run `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
+This will create two Kubernetes files in current directory - `deployment-helloworld.yaml` and `service-helloworld.yaml`.
+
+To deploy your application to Kubernetes run - `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
```

Comment:
I don't know about that. I would actually prefer how it was before. It is just more readable even for people that don't know anything about bash